### ISSUE TITLE
Current Bazarr install method deprecated

### ISSUE THREAD

**Issue by she11sh0cked** [swizzin/swizzin#722]
## Bug description
Swizzin currently installs Bazarr by [cloning the repository](https://github.com/swizzin/swizzin/blob/001bd702300e533ec7213911cc20428732239867/scripts/install/bazarr.sh#L32). As pointed out by @morpheus65535 in https://github.com/morpheus65535/bazarr/issues/1473#issuecomment-882467982 this is now deprecated, resulting in an error.

## Reproduction steps
1. Install package bazarr
2. Go to `/bazarr`
3. See error in bazarr logs

## Installed applications through swizzin
bazarr, deluge, delugeweb, nginx, panel, plex, prowlarr, radarr, sonarr

## Expected behavior
Swizzin should pull the latest asset release.

## Server Info
 - OS: Ubuntu 20.04
 - Arch: x86_64
 - Image Source: Installscript from Kimsufi / OVH

## Is Swizzin up to date?
HEAD is now at 001bd702300e533ec7213911cc20428732239867

## Additional context
See https://github.com/morpheus65535/bazarr/issues/1473

**Comment by flying-sausages** [comment]
Thanks for reporting, good to know. Before I get to this, is there an internal upgrading mechanism for this deployment style?

**Comment by she11sh0cked** [comment]
@flying-sausages You mean something like Sonarr's "Install Latest" button on `/sonarr/system/updates`?

**Comment by flying-sausages** [comment]
Yiis

**Comment by she11sh0cked** [comment]
Doesn't seem like it
https://github.com/morpheus65535/bazarr/blob/31dcd37d52dfb52d091c1131754bba7f1bd94609/frontend/src/System/Releases/index.tsx#L10

**Comment by bakerboy448** [comment]
Appears there is based on the task & noted auto updates

![03754FCA-A8A5-4F9E-9960-65B40DDE7347](https://user-images.githubusercontent.com/55419169/126184988-29e09689-181c-4d9b-8fb7-3c83b41cfc05.jpeg)

**Comment by morpheus65535** [comment]
Yes Bazarr does include an automatic update mechanism unless you use the `--no-update` command line argument to start it: https://github.com/morpheus65535/bazarr/blob/master/bazarr/scheduler.py#L208-L227

**Comment by morpheus65535** [comment]
When the `Update Bazarr` task run, it will check if a new release (or pre-release if development branch is selected in settings) is available. If there's a newer release, it will download the asset in `config/update` then restart itself to apply it.

**Comment by flying-sausages** [comment]
Cool, thanks for the confirm @she11sh0cked 

@morpheus65535 thanks for dropping in. Is replacing the git source files with the content of the release tar a kosher upgrade path that is safe to apply to all installs?

**Comment by morpheus65535** [comment]
> @morpheus65535 thanks for dropping in. Is replacing the git source files with the content of the release tar a kosher upgrade path that is safe to apply to all installs?

Yes it should work perfectly fine. You can remove the `.git` directory just to make it cleaner. :-)

**Comment by flying-sausages** [comment]
@morpheus65535 I'm running into this, I'm assuming it has to do something with the fact we're attempting to launch from a virtualenv? Any pointers to commands I could try attempt to reproduce this situation?

```
Jul 19 17:58:50 m1 systemd[1]: Started Bazarr for test.
Jul 19 17:58:51 m1 bazarr.test[516972]: 2021-07-19 17:58:51,227 - root                             (ffff800be010) :  INFO (init:58) - BAZARR installing requirements...
Jul 19 17:58:51 m1 bazarr.test[516972]: 2021-07-19 17:58:51,749 - root                             (ffff800be010) :  ERROR (init:65) - BAZARR requirements.txt installation result: b"ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.\n"
```

Here's our service file

```systemd
[Unit]
Description=Bazarr for ${user}
After=syslog.target network.target

[Service]
WorkingDirectory=/opt/bazarr
User=${user}
Group=${user}
UMask=0002
Restart=on-failure
RestartSec=5
Type=simple
ExecStart=/opt/.venv/bazarr/bin/python3 /opt/bazarr/bazarr.py
WorkingDirectory=/opt/bazarr
KillSignal=SIGINT
TimeoutStopSec=20
SyslogIdentifier=bazarr.${user}

[Install]
WantedBy=multi-user.target

**Comment by flying-sausages** [comment]
oh my bad it's perms lol

**Comment by liaralabs** [comment]
Fixed by 2e9302cea8d86b9184a55d92a4e9a8f1a96f5dce
